server/controllers/user.controller.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/named
+import { User } from '../models';
+import { generateToken, passwordHash } from '../utils';
+
+/**
+ * User Controller
+ * @package User
+ */
+class UserController {
+  /**
+   * Create a user
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} responseObject
+   */
+  static async create(req, res) {
+    req.body.password = await passwordHash(req.body.password);
+    const newUser = await User.create(req.body);
+    const token = await generateToken({ id: newUser.id });
+    const { firstname, lastname, email } = newUser;
+    const userInfo = {
```

Comment:
You could do well to also destructure line `17` to avoid repetition.
a `try and catch` would suffice here, I know you did a `tryCatch` helper for this which I will be expecting some clarification on.
Good job!

Reply:
Will work on it right away

Reply:
Hey, you destructured and still spelled out things on line `18`. 🤷‍♂️

Reply:
Okay, I think I don't follow. Could you please explain further? 😬

Reply:
After destructuring on line 17, you have a `password` variable. Is there any need to use `req.body.password`?
More so, since you plan to reassign `password`, don't you think you would have issues with `const`?

Reply:
I will reimplement this.